src/index.html

```diff
@@ -22,6 +22,333 @@
+            class="nav__link"
+            data-qa="nav-hover"
+            href="#laptops_and_computers"
+          >
+            laptops & computers
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            href="#gadgets"
+          >
+            gadgets
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            href="#tablets"
+          >
+            tablets
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            href="#photo"
+          >
+            photo
+          </a>
+        </li>
+
+        <li>
+          <a
+            class="nav__link"
+            href="#video"
+          >
+            video
+          </a>
+        </li>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__imac-image"
+          src="./images/imac.jpeg"
+          alt="imac-image"
+        />
+        <div class="card__content">
+          <h3 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="card__product-code">Product code: 195434</p>
+          <div class="stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="stars__review">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <div class="card__price-text">Price:</div>
+            <div class="card__price-tag">$2,199</div>
+          </div>
+          <a
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__imac-image"
+          src="./images/imac.jpeg"
+          alt="imac-image"
+        />
+        <div class="card__content">
+          <h3 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="card__product-code">Product code: 195434</p>
+          <div class="stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="stars__review">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <div class="card__price-text">Price:</div>
+            <div class="card__price-tag">$2,199</div>
+          </div>
+          <a class="card__buy">BUY</a>
+        </div>
```

Comment:
This card is missing the `data-qa="card-hover"` attribute. Please, make sure all cards have this attribute as it might be used for testing purposes.